contrib/devtools/interactive-test-runner.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+# coding=utf-8
+# Copyright (c) 2019 The Unit-e developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+"""
+Runs functional tests while reporting which ones are currently being run.
+
+Tests to be run can be filtered by the arguments passed. Any test that is
+run by the test runner has to match one of the arguments.
+
+Example: Run all "wallet" and "rpc" tests.
+    %(prog)s wallet_ rpc_
+
+All options (--option) that do not target this test runner are passed
+down to the scripts being invoked.
+"""
+
+import argparse
+import os
+import random
+import subprocess
+import sys
+import tempfile
+import time
+
+
+def colorizer(mode):
+    if os.name == 'posix':
+        return lambda s: '\033[' + mode + 'm' + s + '\033[0m'
+    else:
+        return lambda s: s
+
+
+bold = colorizer('1')
+gray = colorizer('1;30')
+red = colorizer('0;31')
+yellow = colorizer('0;33')
+blue = colorizer('0;34')
+
+try:
+    # Make sure python thinks it can write unicode to its stdout
+    '\u2713'.encode('utf8').decode(sys.stdout.encoding)
```

Comment:
Not sure why not directly checking the value of `sys.stdout.encoding` to be `'UTF-8'`, we should avoid using exceptions for flow control, at least when we have other options.

Reply:
I copied it verbatim from `test_runner.py`.

Checking `sys.stdout.encoding` is the same?

Reply:
Changed in 3094b9a707c45a2250e0b7128b91000d3609e422